packages/plugin-react/src/index.ts

```diff
@@ -391,7 +394,7 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
           ...exports.map((name) => `export const ${name} = jsxRuntime.${name}`)
         ].join('\n')
       }
-    }
+    } */
```

Comment:
@aleclarson we plan to merge this PR leaving this as a TODO, I think it is a good idea to revisit it and try to add back the optimization. I would prefer we could do something more generic here but I see the appeal to at least doing it for the jsx runtime that is used everywhere in the app

Reply:
did you try merging #7246 into this PR to see if your issue can be avoided?

Reply:
No, but I was looking into that PR while checking this code. If you have time to try it out, it would be good. If not, I prefer we move forward without the optimization to be able to merge these PRs and we can add it back with #7246 or a similar PR 👍🏼